src/main/java/de/epiceric/shopchest/shop/Shop.java

```diff
@@ -473,4 +476,20 @@ public InventoryHolder getInventoryHolder() {
         return null;
     }
 
+    public void sendbungeeMessage(String player, String message) {
```

Comment:
can this be a private method in the ShopInteractListener? This doesn't seem like it's related to the shop